crates/subspace-node/src/commands/run.rs

```diff
@@ -96,6 +92,7 @@ pub async fn run(run_options: RunOptions) -> Result<(), Error> {
         mut prometheus_configuration,
     } = create_consensus_chain_configuration(consensus, domain_options.is_some())?;
 
+    raise_fd_limit();
```

Comment:
Not sure why this code was moved?

Reply:
This one was the firs tone with logging, bnow initialized into `create_consensus_chain_configuration`